docs/testing.md

```diff
@@ -0,0 +1,64 @@
+## Testing APB Image
+
+### Why would we want to test the image?
```

Comment:
Perhaps an alternative like below 

### Motivation

As the ecosystem of APBs grows we want to facilitate a means for performing a basic sanity check to ensure that an APB is working as the author intended.  The basic concept is to package an integration test with the APB code which will contain all of the needed parameters for a basic provision of service and check that it is up as intended.